bazel/deps.bzl

```diff
@@ -0,0 +1,64 @@
+load(
+  "@bazel_tools//tools/build_defs/repo:git.bzl", 
+  "git_repository",
+  "new_git_repository",
+)
+
+load(
+    "@bazel_tools//tools/build_defs/repo:http.bzl",
+    "http_archive",
+    "http_file",
+)
+
+def azure_storage_cpplite_dependencies():
+
+  if not "boringssl" in native.existing_rules():
```

Comment:
Does bazel have OpenSSL instead of boringssl?

Reply:
Not a lot of sources have bazel build definitions so I copied a few of these from the TensorFlow repo. They use borginssl as a drop in replacement for openssl. Thought it would be easier to follow that, but I can investigate what would be required to use OpenSSL itself